addons/survey/views/survey_templates_user_input_session.xml

```diff
@@ -30,25 +30,39 @@
 
     <template id="user_input_session_open" name="Survey: Open Session">
         <t t-call="survey.user_input_session">
-            <div class="wrap py-3 min-vh-100 align-items-center justify-content-center d-flex o_survey_session_open o_survey_session_manage"
+            <div class="o_survey_session_open o_survey_session_manage position-absolute bottom-0 start-0 end-0 d-flex flex-column justify-content-between gap-md-4 mb-md-3 mx-md-4 fs-3 text-white"
                 t-att-data-survey-access-token="survey.access_token"
                 t-att-data-survey-id="survey.id"
                 t-att-data-is-start-screen="True">
-                <div class="w-75 p-4">
-                    <div class="text-center">
-                        <h1 class="mb-4" t-field="survey.title" />
-                        <h2 class="mb-5 fw-normal">
-                            <span class="text-break">To join: <span class="text-info o_survey_session_copy" t-esc="survey.session_link"/></span>
-                            <input class="o_survey_session_copy_url d-none" type="text" t-att-value="survey.session_link" />
-                        </h2>
-                        <h2 class="fw-normal"><span>Waiting for attendees...</span>
-                            <span class="ms-1 fw-bold o_survey_session_attendees_count" t-esc="survey.session_answer_count" />
-                        </h2>
+
+                <div class="o_survey_session_open_header d-flex flex-column flex-md-row flex-grow-1 flex-md-grow-0 justify-content-between gap-4 p-5 rounded-3 bg-black-25">
+                    <div class="o_survey_session_open_title flex-basis-50">
+                        <h1 t-field="survey.title"/>
+                        <div class="o_survey_session_open_description overflow-auto">
+                            <p t-field="survey.description"/>
+                        </div>
+                    </div>
+                    <div class="d-flex flex-column flex-lg-row gap-4">
+                        <div class="o_survey_session_open_counter d-flex flex-column justify-content-center p-4 rounded-3 bg-black-25 text-center">
+                            <h1><output class="o_survey_session_attendees_count" t-esc="survey.session_answer_count"/></h1>
```

Comment:
My understanding of `<output>` is that it is for any value that is updated by code, here this value is updated every 2 seconds via JS but you're right it is hardly used within odoo, removed. I did use h1 for the count because it shares the same font-size as the title, not the best pick semantically speaking I agree. I actually don't know what to put instead, do you have a suggestion?

Reply:
I've read the mdn page of output and indeed I was using it in a way that is not mandated by mdn. I now wonder why I've been using it wrong for all that time (I've been using it for at least 5 years now, but outside of Odoo). It annoys me because for that `survey.session_link` I need a tag, what do you mandate?